cg/meta/archive/ddn_dataflow.py

```diff
@@ -186,34 +186,39 @@ def auth_header(self) -> Dict[str, str]:
             self._refresh_auth_token()
         return {"Authorization": f"Bearer {self.auth_token}"}
 
-    def archive_folders(self, sources_and_destinations: Dict[Path, Path]) -> bool:
-        """Archives all folders provided, to their corresponding destination, as given by sources and destination parameter."""
-        transfer_data: List[TransferData] = [
-            TransferData(source=source.as_posix(), destination=destination.as_posix())
-            for source, destination in sources_and_destinations.items()
-        ]
-        transfer_request: TransferPayload = TransferPayload(files_to_transfer=transfer_data)
+    def archive_folders(self, sources_and_destinations: List[TransferData]) -> int:
```

Comment:
Function name is misleading with the new implementation as now it returns a job id in addition to starting the transfer request.

Reply:
Would it make it clearer if the response was a Response object? We somehow need to get the job_id but I think it would be worse having "archive_folders_and_get_job_id" as a name than to just return the job_id or response with the current naming.

Reply:
Yes, I think mentioning it in the docstring should be fine in this case.

Reply:
I think archiving a folder is what the function does. The return is just to keep track of it. I Think clarifying the return in the doc string is enough